beacon-chain/db/kv/regen_historical_states.go

```diff
@@ -54,6 +56,10 @@ func (kv *Store) regenHistoricalStates(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	cacheState, err := lru.New(int(params.BeaconConfig().SlotsPerEpoch) * 2)
```

Comment:
can this cache size be at the top of the file as a const

Reply:
It's never used again so we should let this be GC'd after this method returns